pkg/apis/networking/register.go

```diff
@@ -73,9 +73,13 @@ const (
 	// ActivatorServiceName is the name of the activator Kubernetes service.
 	ActivatorServiceName = "activator-service"
 
+	// DisableInternalWildcardCertLabelKey is the label key attached to a namespace to indicate that
+	// a wildcard certificate should be not created for it.
```

Comment:
Will this `internal` label be deprecated in the future? If yes, it should be explained in the comment.

Reply:
I think it should be deprecated in the future. Maybe add a comment to say it will be deprecated to prevent it from being used in the future?

Reply:
We can even name it `DeprecatedDisable...`